src/ast/nodes/TemplateElement.ts

```diff
@@ -1,14 +1,22 @@
 import type * as NodeType from './NodeType';
+import { Flag, isFlagSet, setFlag } from './shared/BitFlags';
 import { type GenericEsTreeNode, NodeBase } from './shared/Node';
 
 export default class TemplateElement extends NodeBase {
-	declare tail: boolean;
 	declare type: NodeType.tTemplateElement;
 	declare value: {
 		cooked: string | null;
 		raw: string;
 	};
 
+	//declare tail: boolean;
+	get tain(): boolean {
```

Comment:
typo:
```suggestion
	get tail(): boolean {
```

This one is a little strange. We don't actually appear to read this value anywhere, so it would be nice to just never set it. Either way we still just pack it into the bitfield so it doesn't really have any overhead.